src/main.vala

```diff
@@ -25,6 +44,12 @@ public static int main (string[] args) {
     Environment.set_application_name (APP_NAME);
     Environment.set_prgname (APP_NAME);
 
+    Bus.own_name (BusType.SESSION, "org.freedesktop.FileManager1", BusNameOwnerFlags.REPLACE,
```

Comment:
I tried this but either I did something wrong or it does not work unless the app is already running.  If not, the app get launched OK but misses the dbus message.  The FM1 service needs to be up and running (in main ()) before the GApplication launches, I think.